### ISSUE TITLE
get error in the post processing effects, refresh error

### ISSUE THREAD

**Issue by zhiqunDLUT** [jscastro76/threebox#316]
i found error in the mapbox-gl custom layer，it got white 
and i test mapbox，use the post processing effects，and the 3d models refresh error
![image](https://user-images.githubusercontent.com/31470793/148342388-290d6122-fa88-4934-9fc8-a96f1193581f.png)

**Comment by jscastro76** [comment]
That branch is not supported anymore, sorry, many things have changed in Mapbox and Three.js since then. 
As explained in #164 there are many reasons for not including post processing effects to Threebox.
Long story short, I decided to not include it in the continuous release for several reasons:

> It affects seriously the performance, despite it depends on the effect, most of them require two or more passes in the EffectComposer which reduces seriously the FPS rate.

> It makes Threebox very heavy. That’s why also Three.js excluded post processing from the core.

> Some effects conflict with Mapbox and the renderer clear order, such as Bloom.

> Once applied affects all the scene. I was very interested in some post processing effects but I wanted to apply only to specific objects on events such as mouseover or selected.

**Comment by zhiqunDLUT** [comment]
thanks a lot&nbsp;
we will go on with it, and find a new way put IOT in threebox and three.js




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jscastro76/threebox"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年1月6日(星期四) 下午3:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jscastro76/threebox] get error in the post processing effects, refresh error (Issue #316)





 
That branch is not supported anymore, sorry, many things have changed in Mapbox and Three.js since then.
 As explained in #164 there are many reasons for not including post processing effects to Threebox.
 Long story short, I decided to not include it in the continuous release for several reasons:
  
It affects seriously the performance, despite it depends on the effect, most of them require two or more passes in the EffectComposer which reduces seriously the FPS rate.
   
It makes Threebox very heavy. That’s why also Three.js excluded post processing from the core.
   
Some effects conflict with Mapbox and the renderer clear order, such as Bloom.
   
Once applied affects all the scene. I was very interested in some post processing effects but I wanted to apply only to specific objects on events such as mouseover or selected.
  
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by jscastro76** [comment]
Indeed I'm already using Threebox in production for thousands of IoT devices, postprocessing wouldn't allow to manage even hundreds because of the passes that reduce significantly the performance
